docs/source/jupyter_lite_config.json

```diff
@@ -68,7 +68,7 @@
       "https://files.pythonhosted.org/packages/py3/t/traitlets/traitlets-5.1.1-py3-none-any.whl",
       "https://files.pythonhosted.org/packages/py3/t/typing_extensions/typing_extensions-4.1.1-py3-none-any.whl",
       "https://files.pythonhosted.org/packages/py3/v/vega-datasets/vega_datasets-0.9.0-py3-none-any.whl",
-      "https://files.pythonhosted.org/packages/py3/w/widgetsnbextension/widgetsnbextension-4.0.0rc1-py3-none-any.whl",
+      "https://files.pythonhosted.org/packages/py3/w/widgetsnbextension/widgetsnbextension-4.0.0rc2-py3-none-any.whl",
```

Comment:
Cannot this also be done with a `../../python/...` path?

Reply:
Great question. I've pushed a commit to try it.

Reply:
Seems to work great!